src/content/docs/opentelemetry/best-practices/opentelemetry-best-practices-resources.mdx

```diff
@@ -135,74 +135,96 @@ with [Amazon CloudWatch Metric Streams](/install/aws-cloudwatch/). This is not
+  The relationships described below that are supported by instrumentation
+  leverage OpenTelemetry semantic conventions.
   Many of these conventions are not yet stable and are still evolving.
   Instrumentation that produces telemetry for the frameworks below vary in
   their level of maturity and conformance to the semantic conventions. As
   such, the instrumentation you use may not emit the attributes required for
   these relationships to work. Please reach out to the maintainers of the
   instrumentation you're using with any questions about the instrumentation's
   maturity with respect to OpenTelemetry's semantic conventions.
-</Callout>
-
-#### ElastiCache (Memcached)
-
-The Amazon CloudWatch Metric Streams integration for
-[Elasticache](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-elasticache-monitoring-integration/)
-generates ElastiCache entities in New Relic. These entities will have the
-following entity tags:
-
-* `aws.elasticache.configurationEndpointAddress` or `aws.elasticache.endpointAddress`
-* `aws.elasticache.configurationEndpointPort` or `aws.elasticache.endpointPort`
-
-If your services use OpenTelemetry instrumentation for Memcached it must emit
-the following attributes for a relationship to be generated:
-
-* `db.system`: This attribute must have the value `memcached`.
-* `net.peer.name`: This attribute must match the corresponding endpoint tag of
-  the ElastiCache entity.
-* `net.peer.port`: This attribute must match the corresponding port tag of the
-  ElastiCache entity.
-
-#### Amazon MQ (ActiveMQ)
-
-The Amazon CloudWatch Metric Streams integration for
-[Amazon MQ](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-mq-integration)
-generates Amazon MQ entities in New Relic. These entities will have the
-following entity tag:
 
-* `aws.mq.endpoint`
-
-If your services use OpenTelemetry instrumentation for ActiveMQ it must
-the following attributes for a relationship to be generated:
-
-* `messaging.system`: This attribute must have the value `activemq`.
-* `server.address`: This attribute must match the corresponding endpoint tag
-  of the Amazon MQ entity.
-* `span.kind`: This relationship is generated from span data where `span.kind`
-  is `producer` or `consumer`.
-
-#### Amazon Managed Kafka (MSK)
-
-The Amazon CloudWatch Metric Streams integration for
-[MSK](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-managed-kafka-msk-integration)
-generates MSK entities in New Relic. These entities will have the
-following entity tag:
-
-* `aws.clusterName` or `aws.kafka.ClusterName`
-* `aws.awsRegion` or `aws.region`
-* `aws.topic` or `aws.kafka.Topic`
-
-If your services use OpenTelemetry instrumentation for MSK it must have
-the following attributes for a relationship to be generated:
+  The relationships not supported by instrumentation require you to manually
```

Comment:
Maybe something to the effect of: "In cases where the relationship is not supported by instrumentation, or where the instrumentation available does not contain the required data, you must manually emit telemetry to signal to New Relic that a relationship should be created."